test_perf.py

```diff
@@ -82,7 +82,7 @@ async def client_receive_async(args, client, partition):
 
 def create_and_open_receive_client(args, partition, clients_arr=None):
     print("Creating and opening receive client:{}".format(partition))
-    creds = SASLPlainCredential(authcid=live_eventhub_config['key_name'], passwd=live_eventhub_config['access_key'])
+    creds = SASLPlainAuth(authcid=live_eventhub_config['key_name'], passwd=live_eventhub_config['access_key'])
```

Comment:
How do you feel about 'Credential' vs 'Auth' here?
Not a topic that needs to be discussed yet - as we can solidify naming much later. Just wondering if you have strong preferences.

Reply:
Depending on the model, I prefer Credential which represents the data it keeps -- Credential used for authentication, Auth is more like a process.